plugins/callback/journald.py

```diff
@@ -0,0 +1,199 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: journald
+    type: notification
+    short_description: Sends events to systemd journal (journald)
+    description:
+      - This callback will report facts and task events to systemd journal (journald)
+    requirements:
+      - whitelisting in configuration
+      - systemd-python (python library)
+'''
+
+import json
+import socket
+import uuid
+from datetime import datetime
+
+import logging
+
+try:
+    from systemd.journal import JournalHandler
+    HAS_SYSTEMD = True
+except ImportError:
+    HAS_SYSTEMD = False
+
+from ansible.plugins.callback import CallbackBase
+
+
+class CallbackModule(CallbackBase):
+    """
+    ansible systemd journal (journald) callback plugin
+    ansible.cfg:
+        callback_plugins   = <path_to_callback_plugins_folder>
+        callback_whitelist = journald
+    and put the plugin in <path_to_callback_plugins_folder>
+
+    Requires:
+        systemd-python
+    """
+
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'notification'
+    CALLBACK_NAME = 'community.general.journald'
+    CALLBACK_NEEDS_WHITELIST = True
+
+    def __init__(self, display=None):
+        super(CallbackModule, self).__init__(display=display)
+
+        if not HAS_SYSTEMD:
+            self.disabled = True
+            self._display.warning("The required systemd-python is not installed. "
+                                  "pip install systemd-python")
```

Comment:
Is this really the recommended way of installing systemd bindings? I think doing it this way it will try to build the bindings from sources which means you would require compilers, devel packages, etc. Maybe instead point here to use the OS package manager commands, eg. `apt-get install python-systemd`

Reply:
It's probably best to mention both ways, i.e. prefer OS package manager, and fall back to pip if that doesn't work for some reason.